nautobot/ipam/tests/test_views.py

```diff
@@ -263,6 +261,30 @@ def test_host_non_modifiable_once_set(self):
         self.assertEqual(200, response.status_code)
         self.assertIn("Host address cannot be changed once created", str(response.content))
 
+    def test_move_ip_addresses_between_namespaces(self):
+        ip_address = self._get_queryset().first()
+        new_namespace = Namespace.objects.create(name="Test Namespace")
+        # Assign model-level permission
+        self.add_permissions("ipam.change_ipaddress")
+
+        # Try GET with model-level permission
+        self.assertHttpStatus(self.client.get(self._get_url("edit", ip_address)), 200)
+
+        form_data = self.form_data.copy()
+        form_data["namespace"] = new_namespace.pk
+        request = {
+            "path": self._get_url("edit", ip_address),
+            "data": post_data(form_data),
+        }
+        response = self.client.post(**request)
+        self.assertEqual(200, response.status_code)
+        self.assertIn(f"Could not determine parent Prefix for {ip_address}! Does it exist?", str(response.content))
+        # Change the parent's namespace to the new namespace that we are going to assign the ip_address to.
+        ip_address.parent.namespace = new_namespace
+        ip_address.parent.save()
+        response = self.client.post(**request)
```

Comment:
This should be a no-op at this point, since you moved the parent and so implicitly already moved the ip_address to the new namespace?

Would a better test be to clone the parent into the new namespace and reparent to it, rather than moving the parent directly?